### ISSUE TITLE
[FEATURE]  Pagination

### ISSUE THREAD

**Issue by lewisushindi** [EddieHubCommunity/awesome-github-profiles#1703]
on the view profiles page, can we add pagination so that users can easily navigate through the profiles instead of having to scroll through a long list

**Comment by Panquesito7** [comment]
Hmm, how would this work exactly? 🤔

**Comment by Shoaib19** [comment]
yes we need pagination so only 10 profiles show on each page if user want to see more profiles, then go to page 2 like google have million searches but only show limited on first page at the bottom it have more pages.
there are several libraries can do pagination one of them i s kaminari also i think bootstrap recently added pagination as well.

**Comment by carlminier** [comment]
2 ways I know that this could be handled. Infinite scroll pagination or Regular pagination. I think infinite scroll would be nice as long as a search functionality also gets added in. I'd like to work on this if someone can add me.

**Comment by Panquesito7** [comment]
CC: @schmelto, what do you think about this? I'm all for it.

**Comment by schmelto** [comment]
> CC: @schmelto, what do you think about this? I'm all for it.
> 
> 

I like the idea of pagination as infinite scroll.

**Comment by Panquesito7** [comment]
If wanted, I could work on this, I'd actually like to work on it. 🙂
